plugins/core/qPCV/PCVCommand.h

```diff
@@ -0,0 +1,35 @@
+#ifndef PCVCOMMAND_H
+#define PCVCOMMAND_H
+
+//##########################################################################
+//#                                                                        #
+//#                      CLOUDCOMPARE PLUGIN                               #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 of the License.               #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#          COPYRIGHT: CloudCompare project                               #
+//#                                                                        #
+//##########################################################################
+
+#include "ccCommandLineInterface.h"
+
+class PCVCommand : public ccCommandLineInterface::Command
+{
+public:
+	PCVCommand();
+
+	~PCVCommand()
```

Comment:
This should be `~PCVCommand() override = default;`

Reply:
Addressed in 3e53c2